apiserver/README.md

```diff
@@ -1084,6 +1084,244 @@ Examples:
   }
   ```
 
+The above example creates a new Ray cluster, executes a job on it and optionally deletes a cluster. As an alternative, the same command allows creating a new job on the existing cluster by referencing it in the payload.
+
+Examples:
+
+Start from creating Ray cluster (We assume here that the [template](test/cluster/template/simple) and [configmap](test/job/code.yaml) are already created).
+
+* Request
+  
+```sh
+curl -X POST 'localhost:8888/apis/v1alpha2/namespaces/default/clusters' \
```

Comment:
can you change the port to 31888 so that all the curl commands are consistent in this document?

Reply:
Done